.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  rubocop:
+    docker:
+      - image: ruby:2.4.4@sha256:99d10d192ac1df0873480b1e0262ff0295a15f1d97e5b7bd111a7bf61a808acc
+    steps:
+      - run: apt-get update
+      - run: apt-get install -y cmake
+      - checkout
+      - run: gem install bundler
+      - run: bundle install
+      - run: bundle exec rubocop
```

Comment:
Should also add one for rspec

Reply:
does that mean we don't need to run both separately?